packages/repository/src/model.ts

```diff
@@ -156,7 +156,11 @@ export abstract class Model {
 
     for (const p in def.properties) {
       if (p in this) {
-        json[p] = (this as AnyObject)[p];
+        let val = (this as AnyObject)[p];
```

Comment:
@raymondfeng What is the difference between `toJSON` and `toObject` function? I couldn't find related document talking about this 2 methods, shall we add some doc to explain the difference and their usage?

Reply:
`toObject()` accepts some options which `toJSON()` does not. I improved the test to show that.

Reply:
thanks got it.